packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -35,7 +35,7 @@ const editorStyles = cx(
     border: `1px solid ${uiColors.gray.light2}`,
     borderRadius: '4px',
```

Comment:
Is likely that we would want to have this radius aligned everywhere we use one, could be a good thing if we can have variables for this too, along the font sizes, spacings, etc ..